presto-main/src/main/java/com/facebook/presto/sql/planner/plan/MarkDistinctNode.java

```diff
@@ -60,7 +60,6 @@ public MarkDistinctNode(@JsonProperty("id") PlanNodeId id,
         return ImmutableSet.<Symbol>builder()
                 .addAll(source.getOutputSymbols())
                 .add(markerSymbol)
-                .addAll(orderBySymbols)
```

Comment:
Instead of removing it here, please do not add it in previous commit (squash the change).